### PR TITLE
Add GeoPandas

### LINKED ISSUE #10387: Request: GeoPandas
### Brand Name

GeoPandas

### Website

https://geopandas.org/en/stable/

### Popularity Metric

- 413,896 Global Rank
- 249,004 Country Rank (USA)
-  4,302 Category Rank

### Official Resources for Icon and Color

svg: https://geopandas.org/en/stable/_static/geopandas_logo_web.svg
color: #222832

### Additional Comments

_No response_

### PR DESCRIPTION
**Pull request by yhmyhm0910**
GeoPandas

**Issue:** closes #10387

**Similarweb rank:** 413,896

### Checklist

- [x] I updated the JSON data in `_data/simple-icons.json`
- [x] I optimized the icon with SVGO or SVGOMG
- [x] The SVG `viewbox` is `0 0 24 24`

### Description

  - Hex value is found in its [official website](https://geopandas.org/en/stable/about/logo.html)
  - Used Inkscape to edit original SVG from [here](https://geopandas.org/en/stable/_static/geopandas_logo_web.svg)
  - No multiple sources used

### CODE REVIEW

**Review by adamrusted — CHANGES_REQUESTED** [review]
I'm seeing some _slight_ differences in the logo, most likely down to precision being too low. Could you re-try with a precision of 4 decimals please? Also, if the SVG source is taken from the homepage, you don't need to link to the exact file. Just using `https://geopandas.org` would be sufficient.

**Comment by yhmyhm0910** [comment]
Is the precision ok? I edited the SVG with Inkscape and changed its size to 24px*24px, fit to the document.

**Comment by PeterShaggyNoble** [comment]
@yhmyhm0910, did you try a precision of 4 first? And what did you use for the optimisation? There are quite a few unnecessary spaces in the `path`.

File: _data/simple-icons.json [hunk 1]
```diff
@@ -5504,6 +5504,12 @@
             "source": "https://www.geocaching.com/about/logousage.aspx",
             "guidelines": "https://www.geocaching.com/about/logousage.aspx"
         },
+        {
+            "title": "Geopandas",
```
Comment (PeterShaggyNoble) [inline _data/simple-icons.json:5508]:
```suggestion
            "title": "GeoPandas",
```

File: icons/geopandas.svg [hunk 2]
```diff
@@ -0,0 +1 @@
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Geopandas</title><path d="M 12.00433,0 C 5.39112,0 0,5.38051 0,12 0,18.61083 5.38245,24 12.00433,24 18.62622,24 24,18.61083 24,12 24,5.38051 18.61755,0 12.00433,0 Z m 9.82882,12 c 0,1.39495 -0.29469,2.72058 -0.8234,3.92491 V 8.07509 c 0.52871,1.19567 0.8234,2.52996 0.8234,3.92491 z m -4.09967,7.97978 V 18.1083 h -2.08884 v 3.01517 c -1.12676,0.45054 -2.35753,0.7018 -3.64031,0.7018 -0.30335,0 -0.60671,-0.0173 -0.91007,-0.0433 V 9.1928 H 9.00542 v 12.16462 c -3.961,-1.27365 -6.83857,-4.99061 -6.83857,-9.36606 0,-5.42383 4.4117,-9.83394 9.83748,-9.83394 2.69556,0 5.14843,1.0917 6.92525,2.85921 v 13.88881 h 0.0607 c -0.39003,0.39855 -0.81474,0.75379 -1.25677,1.07436 z M 15.64464,9.20144 h 2.08884 v 4.33213 h -2.08884 z m -3.28494,5.33719 h 2.08884 v 4.33213 H 12.3597 Z m 0,-8.90686 h 2.08884 V 9.9639 H 12.3597 Z m 3.28494,9.16679 h 2.08884 v 2.04476 H 15.64464 Z M 12.3597,11.22888 h 2.08884 v 2.04477 H 12.3597 Z"/></svg>
```
Comment (PeterShaggyNoble) [inline icons/geopandas.svg:1]:
```suggestion
<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>GeoPandas</title><path d="M 12.00433,0 C 5.39112,0 0,5.38051 0,12 0,18.61083 5.38245,24 12.00433,24 18.62622,24 24,18.61083 24,12 24,5.38051 18.61755,0 12.00433,0 Z m 9.82882,12 c 0,1.39495 -0.29469,2.72058 -0.8234,3.92491 V 8.07509 c 0.52871,1.19567 0.8234,2.52996 0.8234,3.92491 z m -4.09967,7.97978 V 18.1083 h -2.08884 v 3.01517 c -1.12676,0.45054 -2.35753,0.7018 -3.64031,0.7018 -0.30335,0 -0.60671,-0.0173 -0.91007,-0.0433 V 9.1928 H 9.00542 v 12.16462 c -3.961,-1.27365 -6.83857,-4.99061 -6.83857,-9.36606 0,-5.42383 4.4117,-9.83394 9.83748,-9.83394 2.69556,0 5.14843,1.0917 6.92525,2.85921 v 13.88881 h 0.0607 c -0.39003,0.39855 -0.81474,0.75379 -1.25677,1.07436 z M 15.64464,9.20144 h 2.08884 v 4.33213 h -2.08884 z m -3.28494,5.33719 h 2.08884 v 4.33213 H 12.3597 Z m 0,-8.90686 h 2.08884 V 9.9639 H 12.3597 Z m 3.28494,9.16679 h 2.08884 v 2.04476 H 15.64464 Z M 12.3597,11.22888 h 2.08884 v 2.04477 H 12.3597 Z"/></svg>
```

**Review by PeterShaggyNoble — DISMISSED** [review]

**Comment by yhmyhm0910** [comment]
Optimize the logo using [SVGOMG Online Tool](https://jakearchibald.github.io/svgomg/) with a precision of 4

**Review by adamrusted — APPROVED** [review]

**Review by adamrusted — APPROVED** [review]
This is a perfect match, thanks for this @yhmyhm0910! 🎉
Looking forward to more contributions from you! 👀

**Pull request merged**